observation_tickets/Kornephoros.json

```diff
@@ -0,0 +1,15 @@
+{"type": "observation_ticket",
```

Comment:
don't include any of these in commits

Reply:
make sure to remove the compiled python files and other random stuff like weather data too